src/Nri/Ui/Slide/V1.elm

```diff
@@ -0,0 +1,107 @@
+module Nri.Ui.Slide.V1 exposing
+    ( AnimationDirection(..)
+    , withSlidingContents
+    , animateIn, animateOut
+    )
+
+{-| Note: You'll almost certainly want to used keyed nodes if you're
+using this module.
+
+@docs AnimationDirection
+@docs withSlidingContents
+@docs animateIn, animateOut
+
+-}
+
+import Css
+import Css.Animations
+
+
+{-| Slide from right to left or from left to right.
+-}
+type AnimationDirection
+    = FromRTL
+    | FromLTR
+
+
+translateXBy : Float
```

Comment:
My thinking was that in the long run this would be configurable but that I didn't know what the API should be yet -- like, will we also be customizing duration and timing function? probably...?

Reply:
That absolutely makes sense; no need to add customisation yet if we don't know how it's going to be used.